kuma/static/styles/components/wiki/indicators.scss

```diff
@@ -43,7 +43,7 @@ block indicators aka banners
 .warning + .overheadIndicator,
 .overheadIndicator + .overheadIndicator,
 .note + .note {
-    margin-top: ($grid-spacing * -1) + 2;
+    margin-top: ($grid-spacing * -1) + 5px;
```

Comment:
It doesn't make a difference since `$grid-spacing` already has pixels as the unit.